continuous_integration/recipes/dask/meta.yaml

```diff
@@ -46,6 +46,10 @@ test:
     - dask.diagnostics
     - dask.distributed
     - distributed
+    
+  commands:
+    - dask --version
+    - dask info versions
```

Comment:
Note that these tests are expected to fail until https://github.com/dask/dask/pull/9600 is merged in and `dask-core` nightlies are published with the new CLI